.github/component_owners.yml

```diff
@@ -66,3 +66,7 @@ components:
   kafka-exporter:
     - spockz
     - vincentfree
+  span-stacktrace:
+    - jackshirazi
+    - jonaskunz
```

Comment:
we certainly need Jack and Jonas to comment on this PR, to make sure they are on board with supporting it.

Reply:
Yup, totally onboard to support this